Sources/Templates/Mock.swifttemplate

```diff
@@ -481,7 +481,9 @@ class MethodWrapper {
     private func deprecatedMessage(_ preferred: String = "") -> String {
         return "@available(*, deprecated, message: \"This constructor is deprecated, and will be removed in v3.1\(preferred)\")\n\t\t"
     }
-    private var noStubDefinedMessage: String { return "Stub return value not specified for \(method.name). Use given" }
+    private var noStubDefinedMessage: String {
+        return "Stub return value not specified for \(method.name.replacingOccurrences(of: "\t", with: " ")). Use given"
+    }
```

Comment:
👍 Thank you. I will propagate it to other files, as Mock.swifttemplate is generated from Templates folder (we started breaking it into pieces and it is stil wip)